relnotes/libfeatures.bug.md

```diff
@@ -0,0 +1,9 @@
+### Missing versions added to `ocean.LibFeatures`
+
+* `ocean.LibFeatures`
+
+This module was supposed to provide a mean for library writters depending
+on Ocean to conditionally support a feature. However the module hasn't been
+update for a long time, and versions `4.2` to `4.8` were missing in the `v4`
+suite, while no `v5` version (so `5.0` - `5.3`) were present.
+This relese adds all the missing features as well as the current (`5.4`).
```

Comment:
Couple of minors: "mean for library writters" should be "means for library writers" and on the last line "This release"

Reply:
Also, updated!